### PR TITLE
last attempt to stabilize enhance-net

### PR DESCRIPTION
**Pull request by PatWie**
This is the last attempt on my side to stabilized the training.

### CODE REVIEW

**Comment by ppwwyyxx** [comment]
Did the following in https://gist.github.com/ppwwyyxx/6da15f2d9087373635254d4e6fbe4891:
Used the paper's normalization
Fixed VGG scale
Used same learning rate for G and D
Used SeparateGANTrainer

And now I can always obtain a model better than baseline (in all the 3 times I've tried).

**Comment by PatWie** [comment]
closed in favor of 81d5fbd84da905e1a3aa0d168465079da35980dd
